Attempt to unlock mutex that was not locked when running ROS_Package_example

Hi, I'm trying to run [this](https://github.com/fpt-corp/DiRa/tree/master/DiRa_Software/ROS_Package_example) example code with 3 new simulator in [here](https://drive.google.com/drive/u/0/folders/1xVlm9bfXtzGfOPIy4B0X-kFcEoVqdayY?fbclid=IwAR20FAmP0mLKIlXmVzucfAbcS8tHixLRKeFsEymu_LWqC1mAARZB6MXXPTA) on:
- Latest Ubuntu 18.04.3 LTS
- Melodic ROS


I can comfirm that I've connected to the simulator from this log:
```bash
linus@linus-G7-7588:~/catkin_ws$ roslaunch video_stream video_stream.launch
....
 - rosbridge_library.capabilities.unadvertise_service.UnadvertiseService
[INFO] [1571632234.778425]: Rosbridge WebSocket server started on port 9090
[INFO] [1571632255.004101]: Client connected.  1 clients total.
the rosdep view is empty: call 'sudo rosdep init' and 'rosdep update'
[INFO] [1571632255.346756]: [Client 0] Subscribed to team1/set_speed
[INFO] [1571632255.350429]: [Client 0] Subscribed to team1/set_angle
[INFO] [1571632255.353589]: [Client 0] Subscribed to team1/set_camera_angle
```

Here is the log when I try to run `video_stream` package:
```bash
linus@linus-G7-7588:~/catkin_ws$ rosrun video_stream video_stream _image_transport:=compressed
[ERROR] [1571632345.491481355]: Tried to advertise a service that is already advertised in this node [/video_stream/compressed/set_parameters]

(RGB:21999): GLib-GObject-CRITICAL **: 11:32:25.507: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(RGB:21999): GLib-GObject-CRITICAL **: 11:32:25.522: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(RGB:21999): GLib-GObject-CRITICAL **: 11:32:25.551: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(RGB:21999): GLib-GObject-CRITICAL **: 11:32:25.551: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(RGB:21999): GLib-GObject-CRITICAL **: 11:32:25.607: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(RGB:21999): GLib-GObject-CRITICAL **: 11:32:25.628: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
Attempt to unlock mutex that was not locked
Aborted (core dumped)
```
And `lane_detect`:
```bash
linus@linus-G7-7588:~/catkin_ws$ rosrun lane_detect lane_detect _image_transport:=compressed

(View:22023): GLib-GObject-CRITICAL **: 11:32:28.029: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(View:22023): GLib-GObject-CRITICAL **: 11:32:28.060: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(View:22023): GLib-GObject-CRITICAL **: 11:32:28.085: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(View:22023): GLib-GObject-CRITICAL **: 11:32:28.086: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(View:22023): GLib-GObject-CRITICAL **: 11:32:28.133: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(View:22023): GLib-GObject-CRITICAL **: 11:32:28.164: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(View:22023): GLib-GObject-CRITICAL **: 11:32:28.165: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(View:22023): GLib-GObject-CRITICAL **: 11:32:28.166: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(View:22023): GLib-GObject-CRITICAL **: 11:32:28.166: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(View:22023): GLib-GObject-CRITICAL **: 11:32:28.166: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
Attempt to unlock mutex that was not locked
Aborted (core dumped)
```
I've noticed some screen poped up after I ran the above code but then it quick disappear with exception.

Since I don't see any proper traceback ;( I really need you guys help here. Thanks a lot <3

--- a/DiRa_Software/ROS_Package_example/lane_detect/src/main.cpp
+++ b/DiRa_Software/ROS_Package_example/lane_detect/src/main.cpp
@@ -61,7 +61,7 @@ int main(int argc, char **argv)
     car = new CarControl();
 
     if (STREAM) {
-        cv::startWindowThread();
+        // cv::startWindowThread();
 
         ros::NodeHandle nh;
         image_transport::ImageTransport it(nh);

--- a/DiRa_Software/ROS_Package_example/video_stream/src/main.cpp
+++ b/DiRa_Software/ROS_Package_example/video_stream/src/main.cpp
@@ -40,7 +40,7 @@ int main(int argc, char **argv)
     cv::namedWindow("RGB");
     cv::namedWindow("Depth");
     
-    cv::startWindowThread();
+    // cv::startWindowThread();
 
     image_transport::ImageTransport it(nh);
     image_transport::Subscriber sub = it.subscribe("/team1/camera/rgb", 1, imageCallback);

